SWIG/pywin32_swig.patch

```diff
@@ -20,15 +22,38 @@ To build the version of SWIG:
 Huge thanks to Amaury Forgeot d'Arc and Roger Upole for rescuing this!
 
 
---- SWIG1.1p5-orig\Modules\python.cxx	1998-01-03 17:17:40.000000000 +1100
-+++ SWIG1.1p5\Modules\python.cxx	2011-04-29 23:28:30.860296500 +1000
-@@ -61,6 +61,8 @@
+diff -u --recursive ../away/SWIG1.1p5/configure ./configure
+--- ../away/SWIG1.1p5/configure	1998-02-04 10:24:23.000000000 -0500
++++ ./configure	2025-12-01 19:25:19.684043200 -0500
+@@ -638,7 +638,7 @@
+ cat > conftest.$ac_ext <<EOF
+ #line 640 "configure"
+ #include "confdefs.h"
+-main(){return(0);}
++int main(){return(0);}
+ EOF
+ if { (eval echo configure:644: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest; then
+   ac_cv_prog_cxx_works=yes
+@@ -808,7 +808,7 @@
+ cat > conftest.$ac_ext <<EOF
+ #line 810 "configure"
+ #include "confdefs.h"
+-main(){return(0);}
++int main(){return(0);}
+ EOF
+ if { (eval echo configure:814: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest; then
+   ac_cv_prog_cc_works=yes
 
+diff -u --recursive ../away/SWIG1.1p5/Modules/python.cxx ./Modules/python.cxx
+--- ../away/SWIG1.1p5/Modules/python.cxx	1998-01-03 01:17:40.000000000 -0500
++++ ./Modules/python.cxx	2025-12-01 20:04:05.964166100 -0500
```

Comment:
I'm curious, what did you use to generate your patch? I used `git diff` and it doesn't result in file timestamps, which I kinda like in this context.

Reply:
I ended up using `diff` from git bash (MINGW64) which has the same style